### ISSUE TITLE
Investigate switch from webpack to gulp

### ISSUE THREAD

**Issue by claymcleod** [stjude/sjcloud-data-transfer-app#19]
- [x] Investigate [gulp-vueify](https://www.npmjs.com/package/gulp-vueify).

**Estimated Time**: 8 hours

**Comment by siosonel** [comment]
I started this yesterday by installing gulp-vueify, and the install triggered gently-rm, which for some reason removed quasar-framework, perhaps not detecting that it's aliased as "quasar" in webpack. I've never had this issue before when installing other packages.

I followed the documentation for only a little bit, which refers to setting up a vue.config.js. Essentially, this means setting up something similar to webpack's vue-loader, and I got intimidated + lost interest with the work involved with that. I'm thinking we should de-prioritize this task, configuring compilers is usually a rabbit hole that I'd rather not get deep into. 

Instead, I'll work on changing from BrowserSync to webpack-hot-middleware for more reliable and faster dev recompiles.
